README.md

```diff
@@ -251,62 +251,62 @@ define AllFalse: false and false
-            "name" : "SomeTrue",
-            "context" : "Patient",
-            "accessLevel" : "Public",
-            "expression" : {
-               "type" : "And",
-               "operand" : [
-                  { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "true", "type" : "Literal" },
-                  { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "false", "type" : "Literal" }
-               ]
-            }
-         }, {
-            "name" : "AllFalse",
-            "context" : "Patient",
-            "accessLevel" : "Public",
-            "expression" : {
-               "type" : "And",
-               "operand" : [
-                  { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "false", "type" : "Literal" },
-                  { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "false", "type" : "Literal" }
-               ]
-            }
-         }
+  "library" : {
+    "identifier" : { "id" : "TestSnippet", "version" : "1" },
+    "schemaIdentifier" : { "id" : "urn:hl7-org:elm", "version" : "r1" },
+    "usings" : {
+      "def" : [
+        { "localIdentifier" : "System", "uri" : "urn:hl7-org:elm-types:r1" },
+        { "localIdentifier" : "Simple", "uri" : "https://github.com/cqframework/cql-execution/simple", "version" : "1.0.0" }
+      ]
+    },
+    "statements" : {
+      "def" : [ {
+        "name" : "Patient",
+        "context" : "Patient",
+        "expression" : {
+          "type" : "SingletonFrom",
+          "operand" : {
+            "dataType" : "{https://github.com/cqframework/cql-execution/simple}Patient",
+            "type" : "Retrieve"
+          }
+        }
+      }, {
+        "name" : "AllTrue",
+        "context" : "Patient",
+        "accessLevel" : "Public",
+        "expression" : {
+          "type" : "And",
+          "operand" : [
+            { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "true", "type" : "Literal" },
+            { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "true", "type" : "Literal" }
+          ]
+        }
+      }, {
+        "name" : "SomeTrue",
+        "context" : "Patient",
+        "accessLevel" : "Public",
+        "expression" : {
+          "type" : "And",
+          "operand" : [
+            { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "true", "type" : "Literal" },
+            { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "false", "type" : "Literal" }
+          ]
+        }
+      }, {
+        "name" : "AllFalse",
+        "context" : "Patient",
+        "accessLevel" : "Public",
+        "expression" : {
+          "type" : "And",
+          "operand" : [
+            { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "false", "type" : "Literal" },
+            { "valueType" : "{urn:hl7-org:elm-types:r1}Boolean", "value" : "false", "type" : "Literal" }
+          ]
+        }
+      }
       ]}
-   }
+  }
```

Comment:
This indent, on the other hand, looks good. I'm not sure how/why we had a three-space indent before. Thanks for fixing it!